x-pack/plugin/esql/src/main/antlr/EsqlBaseParser.g4

```diff
@@ -132,6 +133,15 @@ deprecated_metadata
     : OPENING_BRACKET metadataOption CLOSING_BRACKET
     ;
 
+metricsCommand
+    : METRICS metricsIdentifier (COMMA metricsIdentifier)* aggregates=fields? (BY grouping=fields)?
+    ;
+
+metricsIdentifier
```

Comment:
Reuse `fromIdentifier` or better yet define a rule to be shared across both commands:

```
indexDefinitions
   : indexDefinition (COMMA indexDefinition)*
   ;

indexDefinition
  : FROM_UNQUOTED_IDENTIFIER
  ;

...
metricsCommand METRICS indexDefinitions aggregates=field? (BY grouping=fields)?
```

Reply:
++ I combined these rules in https://github.com/elastic/elasticsearch/pull/108115/commits/7740ed443c2aef2a0360257370c179300fbcacea.